cypress/integration/pages/liveRadio/tests.js

```diff
@@ -19,13 +19,22 @@ export const testsThatFollowSmokeTestConfig = ({ service, pageType }) =>
         );
       });
 
-      it('should render an H2, which contains/displays a styled subheading', () => {
+      it('should render a paragraph, which contains/displays a styled summary', () => {
         cy.request(`${config[service].pageTypes.liveRadio.path}.json`).then(
           ({ body }) => {
-            if (body.metadata.language === 'en-gb') {
-              const { subheadline } = body.content.blocks[1];
-              cy.get('h2').should('contain', subheadline);
-            }
+            const { text } = body.content.blocks[1];
+            cy.get('[role="main"] p').should('contain', text);
+          },
+        );
+      });
+
+      it('should render an audio player embed', () => {
+        cy.request(`${config[service].pageTypes.liveRadio.path}.json`).then(
+          ({ body }) => {
+            const { id, externalId } = body.content.blocks[2];
+            cy.get(
```

Comment:
What kinda element is this?

Reply:
iframe

Reply:
does something like
```
cy.get(
       `iframe[src="${`/ws/av-embeds/media/${externalId}/${id}`}"]`,
).should('be.visible');
```
work?

Reply:
Gonna try it now

Reply:
I tried that but cypress didn’t like it iirc

Reply:
maybe `cy.get('iframe').get('[src="${`/ws/av-embeds/media/${externalId}/${id}`}"]')`?

Reply:
I think we'd have to split the test due to the difference in `<iframe>` and `<amp-iframe>`

Reply:
```
      it('should render an audio player embed', () => {
        cy.request(`${config[service].pageTypes.liveRadio.path}.json`).then(
          ({ body }) => {
            const { id, externalId } = body.content.blocks[2];
            cy.get(
              `iframe[src="${`/ws/av-embeds/media/${externalId}/${id}`}"]`,
            ).should('be.visible');
          },
        );
      });
    });
```
Works in `cypress/integration/pages/liveRadio/testsForCanonicalOnly.js`

But I can't get it working in the AMP test with `amp-iframe[src="${`/ws/av-embeds/media/${externalId}/${id}`}"]`

Reply:
Update I think I've got it working locally

Reply:
https://github.com/bbc/simorgh/pull/3514/commits/34a3e2bd4d8701c94b2abeec8d54ddf7d3d96aad